### ISSUE TITLE
[Enhancement] Percentage display over each filter bar

### ISSUE THREAD

**Issue by Jaga-Telesin** [joaquimguimaraes/farmr#43]
Would be nice to have percentages displayed above each filter graph bar to let us know the overall distribution of those filter response times.  I find myself doing mental gymnastics to compare them every time I look.

Even just "92%" over a bar could immediately tell me how well that time range was performing, and let me judge it against others (typically the longer ones) quickly.  Displaying it above all simultaneously would be ideal.

I wouldn't want it forced inside a tooltip though, since it's the most important stat that I'm looking for from that graph, right after "do I have a lot of long response times".

**Comment by joaquimguimaraes** [comment]
That's a good suggestion. After writing that graph I realized that perhaps a bar graph is not the best way to display that data, then I made its y-axis scale exponential to improve it a little bit. It would be good to have every filter response time sent to server but realistically speaking that adds a bit of strain to server (we're talking about 9k filters per day per device).

**Comment by Jaga-Telesin** [comment]
Yeah, the exponential scale throws off the eye's evaluation of the bars a bit.  One could have 8500 values and the one next to it (half it's height) could only have 50.  That's where the % display would be truly useful.

No way to calculate that client-side during rendering?
